packages/next/export/worker.js

```diff
@@ -146,7 +146,7 @@ export default async function ({
       }
     }
 
-    if (curRenderOpts.inAmpMode) {
+    if (curRenderOpts && curRenderOpts.inAmpMode) {
```

Comment:
This doesn't handle this line:
https://github.com/zeit/next.js/blob/34adcd5fc9b2820105458a184a1f9d4ca33d18f2/packages/next/export/worker.js#L151

Let's revert this change:
```suggestion
    if (curRenderOpts.inAmpMode) {
```